app/views/sections/items/ItemAlcoholStrengthView.scala.html

```diff
@@ -0,0 +1,67 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import viewmodels.InputWidth._
+@import models.GoodsTypeModel.GoodsType
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukInput: GovukInput,
+        govukButton: GovukButton,
+        h1: components.h1,
+        continueOrExit: components.continueOrExit
+)
+
+@(form: Form[_], action: Call, goodsType: Option[GoodsType])(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("itemAlcoholStrength.title", goodsType.fold(messages("site.item"))(_.toSingularOutput())))) {
+
+    @formHelper(action) {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukInput(
+            InputViewModel(
+                field = form("value"),
+                label = LabelViewModel(HtmlContent(heading))
```

Comment:
I don't think we can have h2 in the label, it causes a failure in the a11y tests

Reply:
I think it's OK for this because it's not in a fieldset.

Reply:
Actually, I think you might be right - I remember they changed the guidance now for it.